backend/handlers/handlers.go

```diff
@@ -379,6 +382,21 @@ func (handlers *Handlers) postAddAccountHandler(r *http.Request) (interface{}, e
 			}
 			configuration = signing.NewAddressConfiguration(scriptType, keypath, jsonAddress)
 		}
+		if jsonCoinCode == "btc" || jsonCoinCode == "ltc" {
+			_, err := btcutil.DecodeAddress(jsonAddress, &chaincfg.MainNetParams)
```

Comment:
you want &ltc.MainNetParams for litecoin, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/83e9f2daf065b60196f80cbec56e143dd00f904f/backend/backend.go#L384

could make a var `params` and switch on jsonCoinCode to assign it to the correct one.

eventually it should be by just calling sth like `backend.Coin(jsonCoinCode).DecodeAddress(...)`

Reply:
Made the switch, marked the eventual feature in a TODO